### ISSUE TITLE
Is there any way to get Hub from a class?

### ISSUE THREAD

**Issue by se** [aspnet/SignalR#320]
I want to get my Hub instance from another class.

Is there any way to do that?

**Comment by davidfowl** [comment]
No. See https://github.com/aspnet/SignalR/issues/182. Hubs are transient and are created when incoming data arrives from the socket. You can get at the hub context but you can't get a hub instance.
